### PR TITLE
[master] add REST api to get adapters information

### PR DESCRIPTION
**Pull request by SharpRazor**
now support get following information of a adapter:
1. adapter_address
2. adapter_status
3. lan_owner
4. lan_name
5. mac_address
6. mac_ip_address: this may not found
7. mac_ip_version: this may not found
And add unit test cases and Doc of REST API

### CODE REVIEW

**Comment by zhshuj** [comment]
This API function works good in my environment, thanks!

**Comment by jichenjc** [comment]
please fix the conflict and sign off the PR...

File: zvmsdk/tests/unit/test_volumeop.py [hunk 1]
```diff
@@ -95,7 +95,6 @@ def test_config_force_detach(self):
     @mock.patch.object(shutil, "rmtree")
     @mock.patch("zvmsdk.volumeop.VolumeConfiguratorAPI._create_file")
     @mock.patch("zvmsdk.dist.LinuxDistManager.get_linux_dist")
-    # @mock.patch("zvmsdk.dist.LinuxDist.get_volume_attach_configuration_cmds")
     @mock.patch("zvmsdk.dist.rhel7.get_volume_attach_configuration_cmds")
```
Comment (jichenjc) [inline zvmsdk/tests/unit/test_volumeop.py]: this is on purpose??

**Comment by jichenjc** [comment]
@SharpRazor please check comments... I think you might want to do some local UT first。。

File: zvmsdk/tests/unit/test_volumeop.py (hunk 1 above)
Comment (SharpRazor) [reply zvmsdk/tests/unit/test_volumeop.py]: I have run the UT test, it succeeded.

Comment (SharpRazor) [reply zvmsdk/tests/unit/test_volumeop.py]: I think there is no need to comment this, so I deleted it.

**Comment by SharpRazor** [comment]
> @SharpRazor please check comments... I think you might want to do some local UT first。。

I ran the test and it passed.

**Pull request merged**